packages/react-app/src/components/auth/LoginForm.js

```diff
@@ -1,42 +1,29 @@
 /* eslint-disable */
 import React, { useEffect, useState } from "react";
-import { Button, Form, Grid, Header, Image, Message, Segment } from "semantic-ui-react";
 import { Link, useHistory } from "react-router-dom";
-//import logo from "../../static/logo.png";
 const index = require("../../lib/e2ee.js");
-const e2e = require("../../lib/e2e-encrypt.js")
-const threadDb = require("../../lib/threadDb.js")
+import {getLoginUser, authorizeUser} from "../../lib/threadDb";
 import test from "./img/test.png";
 import logo from "../../images/logoInverted.png";
 import "./Form.css";
 
 function LoginForm(props) {
   let history = useHistory();
   const [password, setPassword] = useState("");
-  const [email, setEmail] = useState("");
-  const [dbClient,setDBClient] = useState(null)
-  const [identity, setIdentity] = useState(null)
-
-  useEffect(() => {
-    threadDb.init('0x25f77f929eC8bD36ea7Ef06DB98dECD12501').then((result)=>{
-      setDBClient(result.client)
-      setIdentity(result.identity)
-      console.log("UseEffect Login all set!!")
-    })
-  }, []);
 
   async function loginUser() {
-    let loginID = await threadDb.loginUser(email,password,identity,dbClient)
-    if (loginID!==null) {
-      let accounts = await index.getAllAccounts(password);
-      console.log("LoginID:",loginID)
-      const json = {
-        email: email,
-        _id: loginID
+    const dbClient = await authorizeUser(password)
```

Comment:
I think we can also add the wallet login like before just to verify the password and local wallet access.